page-components/expandables.md

```diff
@@ -116,26 +235,105 @@ Standard expandables feature a colored bar, a header, the expand/collapse minico
+            </p>
+        </div>
+    </div>
+    <div class="expandable expandable__padded">
+        <button class="expandable_header expandable_target" title="Expand content">
+            <span class="expandable_header-left expandable_label">
+                Expandable Header 2
+            </span>
+            <span class="expandable_header-right expandable_link">
+                <span class="expandable_cue-open">
+                    Show
+                    <span class="cf-icon cf-icon-plus-round"></span>
+                </span>
+                <span class="expandable_cue-close">
+                    Hide
+                    <span class="cf-icon cf-icon-minus-round"></span>
+                </span>
+            </span>
+        </button>
+        <div class="expandable_content">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit. Neque ipsa voluptatibus soluta nobis unde quisquam
+                temporibus magnam debitis quidem. Ducimus ratione
+                corporis nesciunt earum vel est quaerat blanditiis
+                dolore ipsa?
+            </p>
+        </div>
+    </div>
+    <div class="expandable expandable__padded">
+        <button class="expandable_header expandable_target" title="Expand content">
+            <span class="expandable_header-left expandable_label">
+                Expandable Header 3
+            </span>
+            <span class="expandable_header-right expandable_link">
+                <span class="expandable_cue-open">
+                    Show
+                    <span class="cf-icon cf-icon-plus-round"></span>
+                </span>
+                <span class="expandable_cue-close">
+                    Hide
+                    <span class="cf-icon cf-icon-minus-round"></span>
+                </span>
+            </span>
+        </button>
+        <div class="expandable_content">
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit. Neque ipsa voluptatibus soluta nobis unde quisquam
+                temporibus magnam debitis quidem. Ducimus ratione
+                corporis nesciunt earum vel est quaerat blanditiis
+                dolore ipsa?
+            </p>
+        </div>
+    </div>
+</div>
 {:/nomarkdown}
 
 </div>
 
----
+<h3 class="h4">Expanded</h3>
 
 <div class="content-33 content-first">
 
-#### Hover/Focus/Expanded
+The paragraph copy when expanded should use Avenir paragraph and use the type hierarchy header spacing rules.
 
-* Grey 50% bar
-* Pacific Blue minicon
-
-The header should be addressable by the keyboard to ensure keyboard users can open the expandable. It should match the hover state.
+Expandables can make use of an optional stroke under the title when expanded. The title stroke can help to define the title from the  expanded content for longer sections of text it should be indented left and right to match the text within the expandable section.
```

Comment:
Remove double space. Break sentences between "sections of text" and "it should be indended".